package.json

```diff
@@ -20,7 +20,8 @@
     "icons": "yarn --cwd packages/icons build",
     "dev": "yarn standalone start:mock",
     "test": "yarn --cwd packages/editor test",
-    "test:ci": "lerna run test:ci"
+    "test:ci": "lerna run test:ci",
+    "postinstall": "yarn --cwd packages/protocol"
```

Comment:
Rename this script to what it is doing, e.g. "schema:generate"

Reply:
it's name like this to profit from lifecycle ... if you wan't an explicit name, we'll need to duplicate it:
```
postinstall: generate
generate: yarn --cwd packages/protocol
```

Reply:
we have already a lifecycle hook: prepare

there should be a gernarte script and in the prepare statemant this generate hook should be triggered